backend/mora/common.py

```diff
@@ -464,6 +475,15 @@ def create_bruger_payload(
             },
         ]
 
+    if integration_data:
```

Comment:
Samme spørgsmål som før.

Reply:
jeg er enig - men stadig med den holdning at vi skal lagre en streng i stedet for et objekt